comms/discovery/rpcz/chat_blast_test.go

```diff
@@ -26,6 +26,7 @@ func TestChatBlast(t *testing.T) {
 
 	ctx := context.Background()
 	tx := db.Conn.MustBegin()
+	defer tx.Rollback()
```

Comment:
what's this for?

Reply:
If assertion fails tx is held open and has to timeout... so this makes sure it closes straight away.  Just makes it faster to re-run tests after a failure.